.golangci.yaml

```diff
@@ -333,7 +342,7 @@ linters:
     - maintidx # measures the maintainability index of each function
     - misspell # [useless] finds commonly misspelled English words in comments
     #- nlreturn # [too strict and mostly code is not more readable] checks for a new line before return and branch statements to increase code clarity
-    #- paralleltest # [too many false positives] detects missing usage of t.Parallel() method in your Go test
```

Comment:
IIUC we have only one false positive. I wonder why is this comment so off. Maybe has paralleltest tool improved?